sql/ddl_internal.sql

```diff
@@ -10,11 +10,18 @@
 CREATE OR REPLACE FUNCTION _timescaledb_internal.show_chunks_impl(
     hypertable  REGCLASS = NULL,
     older_than "any" = NULL,
-    newer_than "any" = NULL,
-    caller_name NAME = NULL
+    newer_than "any" = NULL
 ) RETURNS SETOF REGCLASS AS '@MODULE_PATHNAME@', 'ts_chunk_show_chunks'
 LANGUAGE C STABLE PARALLEL SAFE;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.drop_chunks_c_impl(
```

Comment:
we dont use `_c_` for functions anywhere. This should be `drop_chunks_impl`

Reply:
Fixed and actually cleaned up some additional crud in the SQL files that I missed

Reply:
Fixed and removed some additional crud in the SQL files that I missed.